src/coreclr/vm/syncblk.cpp

```diff
@@ -2950,8 +2950,8 @@ void ObjHeader::IllegalAlignPad()
     WRAPPER_NO_CONTRACT;
 #ifdef LOGGING
     void** object = ((void**) this) + 1;
-    LogSpewAlways("\n\n******** Illegal ObjHeader m_alignpad not 0, object" FMT_ADDR "\n\n",
-                  DBG_ADDR(object));
+    LogSpewAlways("\n\n******** Illegal ObjHeader m_alignpad not 0, m_alignpad value: %d\n\n",
```

Comment:
I'd argue we want both in this case. The object to know which one and the padding to know what is wrong.

Reply:
the object is easy to get from the dump.

Reply:
but I'm ok to log both.

Reply:
@Maoni0 If this will always assert somewhere else so the object is easy to find, then I'm fine with that. It just wasn't obvious from this change here that that would be the case.